workspace-eth-connector/src/types.rs

```diff
@@ -45,3 +45,45 @@ pub struct WithdrawResult {
     pub recipient_id: Address,
     pub eth_custodian_address: Address,
 }
+
+#[derive(
+    Default, BorshDeserialize, BorshSerialize, Serialize, Deserialize, Debug, Clone, PartialEq, Eq,
+)]
+pub struct FeeBounds {
```

Comment:
I guess you should import this and the below structs from the `aurora-eth-connector` crate. DRY.

Reply:
fixed in d00dbfa5cd226e2e7a7f72a6d2e38eadd5758a88